common/services/app/civic-uk/scripts.tsx

```diff
@@ -1,3 +1,38 @@
+import { font } from '@weco/common/utils/classnames';
+
+const branding = {
+  removeIcon: true,
+  removeAbout: true,
+  fontFamily: 'Inter, sans-serif',
+};
```

Comment:
More options to explore here: https://www.civicuk.com/cookie-control/documentation/branding